<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    
</body>
<script>
     let p = new Promise((reslove,reject)=>{
        let num = Math.floor(Math.random()*10);
            console.log(num)
            if(num>=5){
                reslove(num+"大于五");
            }else{
                reject(num+"小于5")
            }
     })
     console.log(p)
     // .catch方法用来捕获失败状态的失败信息 当状态为失败时调用
    //  p.catch((err)=>{
    //     console.log(err)
    //  })
    p.then((res)=>{
        console.log(res)
    }).catch(err=>{
        console.log(err)
    })

    // let p1 = new Promise((reslove,reject)=>{
    //     $.ajax({
    //         url:"./data/1.json",
    //         success(res){
    //             reslove(res)
    //         }
    //     })
    // })
    // p1.then(res=>{
    //     console.log(res)
    //     $.ajax({
    //         url:"./data/2.json",
    //         success(data){
    //             console.log(data)
    //             $.hja
    //         }
    //     })
    // })
</script>
</html>